### PR TITLE
Adds tabs to the cling menu

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Adds tabs to the cling menu, and organizes the abilities into the tabs.

## Why It's Good For The Game
Makes it easier to find certain powers

## Images of changes

image

## Testing

See above

## Changelog
:cl:
tweak: Cling Evolution menu now has tabs for different abilities like offensive, defensive, etc.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/changeling/changeling_power.dm [hunk 1]
```diff
@@ -31,6 +31,8 @@
 	var/active = FALSE
 	/// If this power can be used while the changeling has the `TRAIT_FAKE_DEATH` trait.
 	var/bypass_fake_death = FALSE
+	/// Where this ability should be stored in the chanegling menu
```
Comment (Henri215) [inline code/modules/antagonists/changeling/changeling_power.dm:34]:
```suggestion
	/// Where this ability should be stored in the changeling menu
```
image

**Comment by 1080pCat** [comment]
Why/how is swap forms a offensive ability? seems more like it'd be a utility imo

**Comment by spacegeese** [comment]
> Why/how is swap forms a offensive ability? seems more like it'd be a utility imo

duh, because you can swap forms, look in a mirror, and say "jesus fuck i look ugly" to **COMPLETELY OBLITERATE** your enemies!! 😁🤣😁🤣😎😎😎😎😁🤣😂

**Comment by Contrabang** [comment]
> Why/how is swap forms a offensive ability? seems more like it'd be a utility imo

Because almost everything for clings is utility, if its even slightly offensive/defensive I've moved it into a new category.

**Review by Burzah — APPROVED** [review]
LGTM
Just needs the conflicts resolved before merge (again).

**Review by S34NW — APPROVED** [review]

**Pull request merged**